src/openrct2/ride/ride_ratings.c

```diff
@@ -618,15 +618,27 @@ static void ride_ratings_calculate_value(rct_ride *ride)
 		(((ride->excitement * RideRatings[ride->type].excitement) * 32) >> 15) +
 		(((ride->intensity  * RideRatings[ride->type].intensity) * 32) >> 15) +
 		(((ride->nausea     * RideRatings[ride->type].nausea) * 32) >> 15);
+	#ifdef ORIGINAL_RATINGS 
+	sint32 value_orig = value; 
+	#endif
 
 	sint32 monthsOld = gDateMonthsElapsed - ride->build_date;
 
 	// New ride reward
+	if (monthsOld <= 4)
+		value = (value * 3) / 2;   // 1.5x
+
+	else if (monthsOld <= 12) 
```

Comment:
I don't like this sort of formatting. Use brackets for this as the else does not look like its part of the previous if. 

I think we could really do with refactoring this whole function into a table with upper lower month bands and multipliers (+bonuses for original ratings) for these bands. This would mean it would be much easier to see how your version and the previous worked. 

Existing:
age max | multiplier | divisor | bonus
----------|-----------|-------|---------
4 | 1 | 1 | 30
12 | 1 | 1 | 10
63 | 3 | 4 | 0
87 | 1 | 2 | 0
103 | 1 | 4 | 0
119 | 1 | 16 | 0
127 | 1 | 32 | 0
max | 1 | 64 | 0